enterprise/internal/campaigns/webhooks_test.go

```diff
@@ -193,6 +193,192 @@ func testGitHubWebhook(db *sql.DB) func(*testing.T) {
 	}
 }
 
+// Run from integration_test.go
+func testBitbucketWebhook(db *sql.DB) func(*testing.T) {
+	return func(t *testing.T) {
+		now := time.Now()
+		clock := func() time.Time {
+			return now.UTC().Truncate(time.Microsecond)
+		}
+		now = clock()
```

Comment:
I might be missing something, but... is this the idea?

```suggestion
		now := time.Now().UTC().Truncate(time.Microsecond)
		clock := func() time.Time { return now }
```

Reply:
I'll clean that up, it was copied from setup in some old tests and doesn't really make sense anymore